spec/v1/providers/https.spec.ts

```diff
@@ -151,6 +151,7 @@ describe('handler namespace', () => {
     it('should return an empty trigger', () => {
       const result = functions.handler.https.taskQueue.onEnqueue(() => null);
       expect(result.__trigger).to.deep.equal({});
+      expect(result.__endpoint).to.deep.equal({});
```

Comment:
Maybe I'm missing something here. Doesn't _adding_ an `__endpoint` annotation now give you the additional work of checking whether an endpoint is empty before you report it in the manifest? It seems like code that is never run, but would cause some weird crashes if we ever tried.

Reply:
Wow I learned now that trying to deploy a function source with triggers defined using the handler namespace fails for the exact reason you described 😮 . With `__endpoint`, I'll make it so that this isn't the case.